src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -103,125 +106,132 @@ namespace AirflowNetwork {
+        MRR, // Distribution system multiple curve fit power law resistant flow component
+        DMP, // Distribution system damper component
+        ELR, // Distribution system effective leakage ratio component
+        CPD, // Distribution system constant pressure drop component
+        COI, // Distribution system coil component
+        TMU, // Distribution system terminal unit component
+        EXF, // Zone exhaust fan
+        HEX, // Distribution system heat exchanger
+        HOP, // Horizontal opening component
+        RVD, // Reheat VAV terminal damper
+        OAF, // Distribution system OA
+        REL  // Distribution system relief air
     };
 
     // EPlus component Type
     enum class iEPlusComponentType : int
     {
-        Unassigned = 0,
-        SCN = 1, // Supply connection
-        RCN = 2, // Return connection
-        RHT = 3, // Reheat terminal
-        FAN = 4, // Fan
-        COI = 5, // Heating or cooling coil
-        HEX = 6, // Heat exchanger
-        RVD = 7  // Reheat VAV terminal damper
+        Unassigned = -1,
+        SCN, // Supply connection
+        RCN, // Return connection
+        RHT, // Reheat terminal
+        FAN, // Fan
+        COI, // Heating or cooling coil
+        HEX, // Heat exchanger
+        RVD  // Reheat VAV terminal damper
     };
 
     // EPlus node type
     enum class iEPlusNodeType : int
     {
-        Unassigned = 0,
-        ZIN = 1,  // Zone inlet node
-        ZOU = 2,  // Zone outlet node
-        SPL = 3,  // Splitter node
-        MIX = 4,  // Mixer node
-        OAN = 5,  // Outside air system node
-        EXT = 6,  // OA system inlet node
-        FIN = 7,  // Fan Inlet node
-        FOU = 8,  // Fan Outlet Node
-        COU = 9,  // Coil Outlet Node
-        HXO = 10, // Heat exchanger Outlet Node
-        DIN = 11, // Damper Inlet node
-        DOU = 12, // Damper Outlet Node
-        SPI = 13, // Splitter inlet Node
-        SPO = 14  // Splitter Outlet Node
+        Unassigned = -1,
+        ZIN, // Zone inlet node
+        ZOU, // Zone outlet node
+        SPL, // Splitter node
+        MIX, // Mixer node
+        OAN, // Outside air system node
+        EXT, // OA system inlet node
+        FIN, // Fan Inlet node
+        FOU, // Fan Outlet Node
+        COU, // Coil Outlet Node
+        HXO, // Heat exchanger Outlet Node
+        DIN, // Damper Inlet node
+        DOU, // Damper Outlet Node
+        SPI, // Splitter inlet Node
+        SPO  // Splitter Outlet Node
     };
 
     enum class iWPCCntr : int
     {
-        Unassigned = 0,
-        Input = 1,
-        SurfAvg = 2
+        Unassigned = -1,
+        Input,
+        SurfAvg
     };
 
-    int constexpr PressureCtrlExhaust = 1;
-    int constexpr PressureCtrlRelief = 2;
+    enum class PressureCtrl
+    {
+        Unassigned = -1,
+        Exhaust,
+        Relief
+    };
 
-    // DERIVED TYPE DEFINITIONS:
+    enum class AirflowNetworkControl
+    {
+        Unassigned = -1,
```

Comment:
Same thing here, introducing Unassigned is not a good idea since there is no way it can be unassigned. I'll need to review to be certain but I'd prefer not to add this.